### PR TITLE
As user I can create/update/destroy comments

### PR DESCRIPTION
**Pull request by AlinaMoskieva**
review please, @VladislavNikoshnov :)

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+class CommentsController < ApplicationController
```
Comment (AlinaMoskieva) [inline app/controllers/comments_controller.rb:1]:
```ruby
  class ArticleCommentsController
```

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    comment.user = current_user
+    if comment.save
+      respond_with(comment, location: comment.article)
+    else
+      redirect_to article, notice: "Comment could not be created."
+    end
+  end
+
+  def update
+    comment.update_attributes(comment_params)
```
Comment (AlinaMoskieva) [inline app/controllers/comments_controller.rb]: use `update` instead of `update_attributes`

File: app/models/comment.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+class Comment < ActiveRecord::Base
+  paginates_per 5
+
+  belongs_to :user
+  belongs_to :article
+
+  validates :body, presence: true
```
Comment (AlinaMoskieva) [inline app/models/comment.rb]: `validates :user, :article, :body, presence: true`

File: app/policies/comment_policy.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+class CommentPolicy < ApplicationPolicy
+  def create?
+    user
```
Comment (AlinaMoskieva) [inline app/policies/comment_policy.rb]: `!!user`

Comment (AlinaMoskieva) [reply app/policies/comment_policy.rb]: or user.present?

**Review by VladislavNikoshnov — APPROVED** [review]
:ok:

**Pull request merged**